osf/metadata/utils.py

```diff
@@ -49,25 +84,34 @@ def datacite_format_contributors(contributors):
                 })
 
         creators.append({
```

Comment:
nit: It's a little confusing that this still being called "creators" when there's a separate function for formatting "creators" vs "contributors". `contributors_json` to mimic `creators_json` in the above function?